docs/state-diff/README.md

```diff
@@ -0,0 +1,185 @@
+ 🤖⚡ External Computation with Kontrol ⚡🤖
+=================================
+**Injecting Foundry Execution into Proofs**
+
+This folder is an example for including external computation with Foundry into Kontrol proofs.
```

Comment:
Maybe "including results of Foundry execution into the Kontrol initial state"?
I would just phrase it in a bit more details manner. It becomes clear what "external computation with Foundry" is in the next paragraph, but we can probably have a more clear term here too.

Reply:
I like this!

Reply:
I reworked the entire intro! Makes sense to get the point across as to why would we ever want to do all of that!

Reply:
Thank Juan, I think it looks great!